main/midnight/src/ui/uicommandwindow.cpp

```diff
@@ -343,7 +343,7 @@ void uicommandwindow::setupApproachText()
         recruitable_characters[0]);
 }
 
-void uicommandwindow::setCommandTextName(layoutid_t id, bool enabled, id_t characterId)
+void uicommandwindow::setCommandTextName(layoutid_t id, bool enabled, mxid characterId)
```

Comment:
No id_t definition present. Followed 'recruitable_characters' definition (line 343) to find that mxid could be used.

Reply:
doh! My bad!!!!